swap/swap.go

```diff
@@ -37,12 +36,14 @@ import (
 	"github.com/ethereum/go-ethereum/p2p/enode"
 	"github.com/ethersphere/swarm/contracts/swap"
 	contract "github.com/ethersphere/swarm/contracts/swap"
+
 	"github.com/ethersphere/swarm/p2p/protocols"
 	"github.com/ethersphere/swarm/state"
 )
 
 // ErrInvalidChequeSignature indicates the signature on the cheque was invalid
 var ErrInvalidChequeSignature = errors.New("invalid cheque signature")
+var ErrskipDeposit = errors.New("swap-deposit-amount non-zero, but swap-skip-deposit true")
```

Comment:
A little nit-picking, should be `ErrSkipDeposit` (upper case)

Reply:
Changed